src/connector/Cargo.toml

```diff
@@ -59,7 +59,7 @@ easy-ext = "1"
 enum-as-inner = "0.6"
 futures = { version = "0.3", default-features = false, features = ["alloc"] }
 futures-async-stream = { workspace = true }
-gcp-bigquery-client = "0.18.0"
+gcp-bigquery-client = { git = "https://github.com/risingwavelabs/gcp-bigquery-client", rev = "ddd6ba738755efde3c9a922a5e66b2831e2bf75d" }
```

Comment:
Why do we use a fork version?

Reply:
Since there were some `gcp-bigquery-client` dependencies on the underlying version that conflicted with the ones we depend on, I forked and modified it to the version we depend on